doc/index.rst

```diff
@@ -166,10 +166,17 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    introduction/importing_workflows
    introduction/chemistry
    introduction/data
-   introduction/returns
-   introduction/new_opmath
    introduction/logging
 
+.. toctree::
+   :maxdepth: 1
+   :caption: Release news
+   :hidden:
+
+   development/release_notes.md
+   development/deprecations
```

Comment:
I decided not to move the location of these two pages in the codebase, because this location also determines the URL and we have shared these pages extensively over the year.